Environment/covid-19_stats.15m.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/zsh
+
+# <bitbar.title>COVID-19 Stats</bitbar.title>
+# <bitbar.version>v1.1</bitbar.version>
+# <bitbar.author>Wilson Goode</bitbar.author>
+# <bitbar.author.github>wilsongoode</bitbar.author.github>
+# <bitbar.desc>Displays stats of US COVID-19 cases, with a submenu for user-defineable States. Can also be configured to show the top n states.</bitbar.desc>
+# <bitbar.image>https://github.com/wilsongoode/covid-bitbar/blob/master/screens/covid-19_top15_states.png</bitbar.image>
```

Comment:
Use the raw link of https://github.com/wilsongoode/covid-bitbar/raw/master/screens/covid-19_top15_states.png

Reply:
thank you, updated!